client/src/LandingPage.js

```diff
@@ -197,8 +216,8 @@ const LandingPage = () => {
                         width: '100%',
                         minHeight: '100vh',
                         padding: '4rem',
-                        background:
-                            'linear-gradient(116.82deg, #0288d1 0%, #1976d2 100%)',
+                        background: 'linear-gradient(116.82deg, rgba(2, 136, 209, 0.75) 0%, rgba(25, 118, 210, 0.75) 100%), url(/assets/landing-page-imgs/earth-digital.jpg)',
```

Comment:
@vishal-codes You can change the image url and properties here.

Reply:
Cool got it